Sources/Public/DotLottie/DotLottie.swift

```diff
@@ -0,0 +1,153 @@
+//
+// DotLottie.swift
+// LottieFiles
+//
+// Created by Evandro Harrison Hoffmann on 27/06/2020.
+// Copyright © 2020 LottieFiles. All rights reserved.
+//
+
+import Foundation
+import ZIPFoundation
+
+// MARK: - DotLottie
+
+/// Detailed .lottie file structure
+public final class DotLottie {
+
+  // MARK: Lifecycle
+
+  /// Loads `DotLottie` from `URL` containing .lottie file.
+  ///
+  /// - Parameters:
+  ///  - url: URL to .lottie file
+  ///  - Returns: Deserialized `DotLottie`. Optional.
+  init(url: URL) throws {
```

Comment:
Is it possible to make these initializers `async`?

Reply:
These initializers to a lot more work than `LottieAnimation.init` so it would be nice to help folks avoid calling this on the main thread